public/index.html

```diff
@@ -50,21 +50,10 @@
         <script src="/lib/autofill-directive/autofill-directive.js"></script>
         <script src="/lib/underscore/underscore-min.js"></script>
         
-        <script src="/modules/app.js"></script>
-        <script src="/modules/base/controllers/main.js"></script>
-        <script src="/modules/base/controllers/login.js"></script>
-        <script src="/modules/base/controllers/navbar.js"></script>
-        <script src="/modules/base/controllers/signup.js"></script>
-
-        <script src="/modules/base/services/User.js"></script>
-        <script src="/modules/base/services/authService.js"></script>
-        <script src="/modules/base/services/menus.js"></script>
-        <script src="modules/base/directives/confirmMessage.js"></script>
-        <script src="modules/base/services/authService.js"></script>
-        <script src="modules/base/services/menus.js"></script>
-        <script src="modules/base/interceptors/authInterceptor.js"></script>
-        <!-- //===== mean-cli hook =====// -->
-
+        <!-- App scripts -->
+        <script src="/dist/scripts/all.min.js"></script>
```

Comment:
So how do you solve the `index.html` file? using a pre-compiling tool? Something like:

```
<!-- if(production) -->
<script src="/dist/scripts/all.min.js"></script>

<!-- if(dev) -->
[all scripts list goes here]
```